docs/pages/project/contributing/contributing-ui.md

```diff
@@ -75,6 +75,14 @@ The designs in this specification should result in enabling:
 
 ## Setup
 
+### Node Version Recommendations 
+We recommend to use LTS versions of node:
+
+- Node 16 LTS: https://nodejs.org/download/release/v16.19.0/ \
+- Node 18 LTS: https://nodejs.org/en/
+
+Although, the script supports other node versions as well like node15 and node17, it is recommended to use node version above 15 and mainly the use of LTS versions like 16 and 18 is highly encouraged.
```

Comment:
```suggestion
Although, the script supports other node versions as well like node15 and node17, it is recommended to use node version above 15 and mainly the use of node LTS v18 is highly encouraged.
```

Reply:
How about this?
The script supports other node versions as well, including node15 and node17, but is recommended to use node versions above 15 and especially node LTS v18.

Reply:
```suggestion
The script supports other node versions as well, including node15 and node17, but is recommended to use node versions above 15 and especially node LTS v18.
```